frontend/packages/ez-transactions/src/features/signless-transactions/hooks/index.ts

```diff
@@ -8,13 +9,16 @@ import {
 } from './use-signless-send-message';
 import { useIsAvailable } from './use-is-available';
 import { useRandomPairOr } from './use-random-pair-or';
+import { useProgramMetadata } from './use-program-metadata';
 
 export {
+  useCountdown,
   useCreateMetadataSession,
   useCreateSailsSession,
   useSignlessSendMessage,
   useSignlessSendMessageHandler,
   useIsAvailable,
+  useProgramMetadata,
```

Comment:
Since we'are not moving to `gear-js` repo, we can continue to reuse hooks from a `@dapps-frontend` package.